Andela-Ray/PropertyPro-lite
# #167101288 Add API documentation to API

**What does this PR do?**
Adds documentation to our API endpoint

**Description of Task to be completed?**
Add API documentation that can be accessed directly from the API URL

**How should this be manually tested?**
Go to the API endpoint URL at: https://andelaray-propertypro-lite.herokuapp.com/
and you will see the link to the API documentation hosted at Swagger there

**What are the relevant pivotal tracker stories?**
#167101288
